metadata-ingestion/tests/unit/api/source_helpers/test_auto_system_metadata.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime
+
+from freezegun import freeze_time
+
+from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.ingestion.api.common import PipelineContext
+from datahub.ingestion.api.source_helpers import AutoSystemMetadata
+from datahub.ingestion.run.pipeline_config import (
+    FlagsConfig,
+    PipelineConfig,
+    SourceConfig,
+)
+from datahub.metadata._schema_classes import SystemMetadataClass
+
+FROZEN_TIME = datetime.fromisoformat("2023-01-01 00:00:00+00:00")
+FROZEN_TIME_TS = int(FROZEN_TIME.timestamp() * 1000)
+DUMMY_URN = "urn:li:dataset:(snowflake, test.dummy, PROD)"
```

Comment:
are the blanks on purpose?

```suggestion
DUMMY_URN = "urn:li:dataset:(snowflake,test.dummy,PROD)"
```